shared/oae/api/oae.api.util.js

```diff
@@ -1656,12 +1656,16 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
                 return '';
             } else {
 
+                // Escape asterisks instead of treating them as italics markers
+                input = input.replace(/\*/g, "\\*").replace(/\\\\/g, "\\");
```

Comment:
I think this is _easier_, since that would require a negative lookbehind, which doesn't exist in JS?